src/System.Management.Automation/engine/MshCommandRuntime.cs

```diff
@@ -2254,7 +2254,7 @@ internal void ThrowIfWriteNotPermitted(bool needsToWriteToPipeline)
             {
                 // Only generate these exceptions if a pipeline has already been declared as the 'writing' pipeline.
                 // Otherwise, these are probably infrastructure messages and can be ignored.
-                if (this.PipelineProcessor._permittedToWrite != null)
+                if (this.PipelineProcessor?._permittedToWrite != null)
```

Comment:
Could you add comment why did change was made?

Reply:
The [line 2249](https://github.com/PowerShell/PowerShell/blob/master/src/System.Management.Automation/engine/MshCommandRuntime.cs#L2249) has 

if (this.PipelineProcessor == null  || _thisCommand != this.PipelineProcessor._permittedToWrite ...

When I debug the code, i saw bunch of nullref exceptions. 
Yes the change has nothing to do with the cursor position. But it's good to get rid of it.

Reply:
@adityapatwardhan In this case I don't think any comments are needed.